server/@types/express/index.d.ts

```diff
@@ -53,6 +53,7 @@ declare module 'express-session' {
     qualificationDetailsForm: QualificationDetailsForm
     additionalTrainingForm: AdditionalTrainingForm
     prisonerContexts: PrisonerContexts
+    educationDto: EducationDto
```

Comment:
Do we need this on the session? 
I'm not sure we do, do we? Because your middleware function adds it to `res.locals` (which I think is the right approach)

Reply:
Removed.